vagga.yaml

```diff
@@ -0,0 +1,36 @@
+minimum-vagga: v0.7.0
+containers:
+  pip-freeze:
+    setup:
+    - !Alpine v3.4
+    - !Py3Install
+      - pip
+      - 'Django >=1.8'
+      - 'gunicorn >=19.6.0'
+      - 'whitenoise >=3.2.3'
+      - 'django-environ >=0.4.1'
+    - !Sh pip freeze > requirements.txt
+  django:
+    setup:
+    - !Alpine v3.4
+    - !PipConfig
+      dependencies: true
+    - !Py3Requirements requirements.txt
+  alpine:
+    setup:
+    - !Alpine v3.4
+    - !PipConfig
+      dependencies: true
+    - !BuildDeps [postgresql-dev]
+    - !Install [python, python3, libpq]
```

Comment:
A proper Python interpreter is usually automatically installed when tags like `!Py3Requirements` or other python-related are used.